content/realtime/metachannels.textile

```diff
@@ -0,0 +1,111 @@
+meta_keywords: "Ably, metachannels, metadata, channel metadata, lifecycle events, connection lifecycle, channel lifecycle."
+section: realtime
+index: 5
+jump_to:
+  Help with:
+    - Overview#overview
+    - Permissions#permissions
+    - Subscribing to metachannel events#subscribing-to-metadata-events
+---
+
+h2(#overview). Overview
+
+Metachannels are a namespace of channels which all start with the @[meta]@ qualifier, differentiating them from regular channels. An example of a metachannel is @[meta]channel.lifecycle@.
+
+There are a number of metachannels available:
+
+- [meta]log := This metachannel broadcasts error messages that you would not otherwise have any way of receiving (with the exception of errors delivering push notifications, see below). For example, if Ably receives an error response when delivering a reactor webhook to your endpoint, Ably sends a message on the log metachannel with information about the error. Errors where the client can be directly notified are not sent on the log metachannel. For example, if a client attempts to publish a message but exceeds a channel rate limit, the client can be notified by an error callback "passed to publish":/realtime/channels#publish.
+- [meta]log:push := This metachannel is similar to @[meta]log@, but only for errors that happen during delivery of push notifications.
+- [meta]channel.lifecycle := This metachannel publishes messages about channel lifecycle.
+- [meta]connection.lifecycle := This metachannel publishes messages about the lifecycle of realtime connections.
+
+h2(#permissions). Permissions
+
+An Ably key has a "capability":/core-features/authentication#capabilities-explained that lists available resources and, for any given resource, a set of permitted operations. The wildcard resource '@*@' will match any regular channel name.
+
+In order to grant permission to a user to access a "metachannel":#metachannels, the resource name or names in the @capability@ must include the @[meta]@ qualifier explicitly. If you are using an API Key, you can "set capabilities in your dashboard":https://support.ably.com/solution/articles/3000073773.
+
+If you are making use of "tokens":/core-features/authentication#token-authentication, you specify capabilities within the token. The following are examples of capabilities that will permit access to a metachannel using tokens:
+
+minimize. See an example of a capability allowing subscription to all metachannels
+  ```[json]
+  {"[meta]*":["subscribe"]}
+  ```
+
+    The above capability allows for the key to @subscribe@ to any @meta@ channel. The wildcard @*@ indicates anything can follow the @[meta]@ claim, so an example of a valid channel would be @[meta]log@. However, this capability does not allow for any other actions to be performed on the metachannels, nor does it allow the key to do anything with any non-metachannels.
```

Comment:
Stray whitespace? (here and below)

Reply:
Yes, they are minimized sections, but I've tabbed back so the start of text aligns with the codeblock.